config/routes.rb

```diff
@@ -121,6 +121,11 @@
     get "/", to: "simulator_old#new", as: "simulator_old_new"
     get "/embed/:id", to: "simulator_old#embed"
   end
+
+  scope "/testbench" do
+    get "/", to: "test_sets#new", as: "testbench_new"
+    post "/create", to: "test_sets#create", as: "testbench_create"
```

Comment:
I think you can just use resources for this, it will automatically add all CRUD routes.